code/datums/gamemode/factions/bloodcult/bloodcult.dm

```diff
@@ -299,6 +299,7 @@ var/veil_thickness = CULT_PROLOGUE
 				..()
 				command_alert(/datum/command_alert/bloodstones_raised)
 				new_obj = new /datum/objective/bloodcult_bloodbath
+				ticker.StartThematic("endgame")
```

Comment:
This is literally the exact opposite of what I told you to do. Implement this in an OOP way.

Reply:
I think I got it chief